packages/paste-core/components/switch/src/SwitchHelpText.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import type {SwitchHelpTextProps} from './types';
+import {HelpText} from '@twilio-paste/help-text';
+import {Box, safelySpreadBoxProps} from '@twilio-paste/box';
+
+const SwitchHelpText = React.forwardRef<HTMLSpanElement, SwitchHelpTextProps>(
+  ({element = 'SWITCH_HELP_TEXT_WRAPPER', children, helpTextId, ...props}, ref) => {
+    return (
+      // @ts-ignore TODO:no 50px tokens
+      <Box {...safelySpreadBoxProps(props)} element={element} marginLeft="50px" ref={ref}>
```

Comment:
We don't have any tokens for this margin (it aligns the help text with the label text). Is the ts-ignore here ok?

Reply:
Not ideal. A question for design @twilio-labs/design-systems-pd if we have a "Good enough" token for here?

Reply:
this is resolved by simon's commit